ndk-build/src/manifest.rs

```diff
@@ -112,6 +114,7 @@ impl Default for Activity {
             launch_mode: None,
             name: default_activity_name(),
             orientation: None,
+            exported: None,
```

Comment:
We should default this to `true` for the activity(ies) with intent that advertise `LAUNCHER` - at least for the one that is added by default with:

https://github.com/rust-windowing/android-ndk-rs/blob/7f6ce6fe0fd33a43aa606a20915b86301a2e0f3b/ndk-build/src/manifest.rs#L102-L106

and

https://github.com/rust-windowing/android-ndk-rs/blob/7f6ce6fe0fd33a43aa606a20915b86301a2e0f3b/ndk-build/src/manifest.rs#L124-L147

Reply:
Android prior to 12 will already consider it true if missing. They made the choice to make it safer by forcing the devs to explictly set it. I think we should not reintroduce the "automatically true when `IntentFilter` is present".

Reply:
We provide a default `android.app.NativeActivity` activity with the default `LAUNCHER` target when no other `MAIN` intent action is specified to allow users to easily build single-executable Android activities with Rust (only a single `android.app.NativeActivity` can exist anyway - and even our manifest representation only contains a singular `activity: Activity`) without having to copy this verbose config into `Cargo.toml` for every project.

For the same purpose, if the user left `exported` unset (`None`) in their manifest we should do our due-diligence to set it to `true` for the default activity so that their app functions consistently across Android 12 and <= 11.